tests/main/parallel-install-desktop/task.yaml

```diff
@@ -15,7 +12,7 @@ execute: |
     for instance in foo longname; do
         echo "Sideload same snap as different instance named basic-desktop+$instance"
         expected="^basic-desktop_$instance 1.0 installed\$"
-        install_local_as basic-desktop "basic-desktop_$instance" | MATCH "$expected"
+        "$TESTSTOOLS"/snaps-state install-local-as basic-desktop "basic-desktop_$instance" | MATCH "$expected"
```

Comment:
Given that now we can pass options to snap install, we could drop install-local-as and just pass --name? Maybe something to think of once all the pieces have landed.

Reply:
@bboozzoo it is good idea. I'll propose that once the migration is finished